src/lib/metric_ingest.py

```diff
@@ -350,7 +373,7 @@ def create_entity_id(service_name: str, service_dimensions: List[Dimension], tim
     return entity_id
 
 
-def convert_point_to_ingest_line(
+def _convert_point_to_ingest_line(
```

Comment:
Nice of you to include some code cleaning! 😄

Reply:
We can do minor things like this in every PR, when we spot them